tests/test-integration/src/test/resources/combined/WEB-INF/file

```diff
@@ -0,0 +1 @@
+This is in the second WEB-INF dir.
```

Comment:
EOLN

Reply:
This is intentional so we don't have to check for newline in the response for the test.